guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,268 @@
+
+### Sending and Receiving Ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+When a user sends an ecash token to another user, the wallet verifies the mint's signature to confirm the token's authenticity. The mint then checks its database to ensure the token has not been previously spent. If the token is valid, it is destroyed and replaced with a new token, which is then issued to the recipient. This cycle of destroying the old token and issuing a new one prevents double spending while maintaining user privacy and security.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   alt-text = "An illustration comparing how on-chain bitcoin, lightning, and ecash handle the sending of bitcoin."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of bitcoin-backed ecash is that the tokens are just strings of text, which are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
+
+#### Text Message, Such as Email or SMS:
+Users can send ecash by copying and pasting the token string. This is particularly useful for online transactions or when using text based communication platforms.
+
+#### Bluetooth Transmission:
+Most applications allow users to share a token via Bluetooth. However, more comprehensive Bluetooth sending and receiving functionalities have not yet been fully developed.
+
+#### QR Codes:
+Unlike base chain and Lightning Network QR codes, which only provide directions for where to send Bitcoin, ecash tokens can be embedded within a QR code itself. This means that a user can claim the ecash token by simply scanning the QR code. This method is particularly useful for in person transactions or quick transfers. QR redemption also enables physical bearer assets like paper notes.
+
+
+---
+
+### [Design Best Practices]({{ '/guide/how-it-works/ecash/design-best-practices/' | relative_url }})
```

Comment:
I’m definitely open to better placement ideas. I thought it fit well here because it follows the explanation of how it works from the user’s perspective and comes before the trade-off sections. It seems helpful for the user to first understand how it works, then immediately see visual examples of some of the features and functionalities they just read about.

Reply:
I'd agree with @GBKS. I would probably move it to the end of the page or scrap it entirely.

Reply:
Agreed. I decided to scrap it. The design best practices are fine in their own section. I don't think they need to be linked in this introduction page.